docs/example/public-hub-content.mdx

```diff
@@ -58,7 +58,7 @@ export SOURCE_REPO_REF="{ref}" # e.g. main
 # The Base64 encoded private ssh key used by the populator to download manifests. It needs the read-only access.
 export SOURCE_SSH_KEY="{key}"
 
-capact install --capact-overrides="hub-public.populator.enabled=true,hub-public.populator.manifestsLocation.repository=${SOURCE_REPO_URL},hub-public.populator.manifestsLocation.branch=${SOURCE_REPO_REF},hub-public.populator.manifestsLocation.sshKey=${SOURCE_SSH_KEY}"
+capact install --capact-overrides="hub-public.populator.enabled=true,hub-public.populator.manifestsLocations[0].repository=${SOURCE_REPO_URL},hub-public.populator.manifestsLocations[0].branch=${SOURCE_REPO_REF},hub-public.populator.manifestsLocations[0].sshKey=${SOURCE_SSH_KEY}"
 ```
 
```

Comment:
It would be good to mention in this doc that we can use multiple manifest sources. I would suggest adding such note to every Tab:

```suggestion

> **NOTE**: You can provide multiple manifest sources by setting overrides using different indexes of the `hub-public.populator.manifestsLocations`.

```